spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -788,21 +788,26 @@ def name_id_version(format_urn)
             user_fully_authenticated: true,
           })
         expect(@analytics).to receive(:track_event).
-          with('SAML Auth', {
-            success: true,
-            errors: {},
-            error_details: nil,
-            nameid_format: Saml::Idp::Constants::NAME_ID_FORMAT_PERSISTENT,
-            authn_context: [Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF],
-            authn_context_comparison: 'exact',
-            requested_ial: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
-            service_provider: sp1_issuer,
-            endpoint: "/api/saml/auth#{path_year}",
-            idv: false,
-            finish_profile: false,
-            request_signed: true,
-            matching_cert_serial: saml_test_sp_cert_serial,
-          })
+          with(
+            'SAML Auth',
+            hash_including(
+              {
+                success: true,
+                errors: {},
+                error_details: nil,
+                nameid_format: Saml::Idp::Constants::NAME_ID_FORMAT_PERSISTENT,
+                authn_context: [Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF],
+                authn_context_comparison: 'exact',
+                requested_ial: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
+                service_provider: sp1_issuer,
+                endpoint: "/api/saml/auth#{path_year}",
+                idv: false,
+                finish_profile: false,
+                request_signed: true,
+                matching_cert_serial: saml_test_sp_cert_serial,
+              },
```

Comment:
we can drop the optional `{`'s right?
```suggestion
              success: true,
              errors: {},
              error_details: nil,
              nameid_format: Saml::Idp::Constants::NAME_ID_FORMAT_PERSISTENT,
              authn_context: [Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF],
              authn_context_comparison: 'exact',
              requested_ial: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
              service_provider: sp1_issuer,
              endpoint: "/api/saml/auth#{path_year}",
              idv: false,
              finish_profile: false,
              request_signed: true,
              matching_cert_serial: saml_test_sp_cert_serial,
```

Reply:
Thanks, I didn't catch that.